_docs/_latest/setup/index.md

```diff
@@ -27,9 +27,9 @@ to the latest version.
 
 Deploy Forseti manually on GCP.
 
-**[Deploy Forseti Security on Kubernetes]({% link _docs/latest/setup/kubernetes.md %})**
+**[Deploy Forseti Security on Google Kubernetes Engine]({% link _docs/latest/setup/forseti-on-gke.md %})**
 
-Proof-of-Concept of Forseti running on Kubernetes.
+Deploy Forseti on Google GKE
```

Comment:
Should add (**BETA**) at the end: 

Deploy Forseti on Google GKE (**BETA**) 


(**BETA**) should be bold!

Reply:
Additionally, Should probably change `Google GKE` to either `GKE` or `Google Kubernetes Engine` as it seems redundant (Google Google Kubernetes Engine)